docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/FastTreeTemplate.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.Regression
+{
+    public static class FastTree2
```

Comment:
This file is auto generated from `FastTreeTemplate.tt` and `RegressionTemplate.txt`.